integration-cli/docker_cli_push_test.go

```diff
@@ -60,7 +60,32 @@ func (s *DockerRegistrySuite) TestPushMultipleTags(c *check.C) {
 
 	dockerCmd(c, "tag", "busybox", repoTag2)
 
-	dockerCmd(c, "push", repoName)
+	out, _ := dockerCmd(c, "push", repoName)
```

Comment:
The error is checked by `dockerCmd`, the `_` is the status which is irrelevant here